### ISSUE TITLE
Question about R_1 calculation.

### ISSUE THREAD

**Issue by ZEXUANW** [openproblems-bio/task_grn_inference#28]
For evaluation approach R1 (regulatory links as features), there are two metrics: R1_GRN (evaluated only on genes present in both the GRN and the test dataset) and R1_all (evaluated on all genes in the test dataset). For example, in the OPSCA dataset, the training set has 22,787 genes, whereas the test set has 15,215 genes, resulting in approximately 13,000 genes in common. The Pearson correlation-based GRN constructed from the training set is 22,787 × 22,787, but the evaluation dataset includes only 15,215 genes. How should we handle this mismatch practically? Specifically, the current manuscript only discusses the scenario of genes missing from the GRN; how should we address the complementary issue—genes present in the GRN but missing in the evaluation dataset?

In this case seems R1 grn and R1 all are same? Thanks!

**Comment by pgniewko** [comment]
@janursa A follow up question on a related issue. 

Genes that are part of the test set can be missing from the predicted GRNs either (i) due to the modeling strategy (and the evaluation metric penalizes them for this) or (ii) because they are absent from the training set, in which case the model cannot make predictions for them. In the latter case, the genes that are present in the test set but absent from the training set contribute to bias in the R1_all metric. Since approximately ~2000 out of ~15000 genes in the test set (as noted above) are missing from the training set, this bias may be non-negligible. The manuscript is not clear on whether these disjoint data points are included in the evaluation. Could you please clarify?
